web/src/components/admin/settings/AdminUsersTab.vue

```diff
@@ -118,14 +118,18 @@ const { doSubmit: saveUser, isLoading: isSaving } = useAsyncAction(async () => {
 
   if (isEditingUser.value) {
     await apiClient.updateUser(selectedUser.value);
+    notifications.notify({
```

Comment:
@lafriks Thanks for fixing those messages. Could you maybe also correct the spacing of the admin badge / buttons for me 😅 : 

![image](https://user-images.githubusercontent.com/6918444/226207341-23689663-056d-471c-92cd-c24a43787b27.png)

Reply:
they do look ok to me:
![attels](https://user-images.githubusercontent.com/165205/226207968-95691e8b-ca1f-4b62-8a99-9ed854216d09.png)